charts/gluu-all-in-one/templates/secret.yaml

```diff
@@ -53,3 +53,33 @@ type: Opaque
 data:
   {{ .Values.cnConfiguratorKeyFile | base }}: {{ .Values.cnConfiguratorKey | b64enc }}
 {{- end }}
+
+---
+
+{{ if .Values.configmap.cnSqlSslEnabled -}}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ .Release.Name }}-sql-ssl
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: {{ .Release.Name }}-{{ include "flex-all-in-one.name" . }}-aio
+{{ include "flex-all-in-one.labels" . | indent 4 }}
+{{- if .Values.additionalLabels }}
+{{ toYaml .Values.additionalLabels | indent 4 }}
+{{- end }}
+{{- if or (.Values.additionalAnnotations) (.Values.customAnnotations.secret) }}
+  annotations:
+{{- if .Values.additionalAnnotations }}
+{{ toYaml .Values.additionalAnnotations | indent 4 }}
+{{- end }}
+{{- if .Values.config.customAnnotations.secret }}
+{{ toYaml .Values.customAnnotations.secret | indent 4 }}
+{{- end }}
+{{- end }}
+type: Opaque
+data:
+  sql_cacert.pem: {{ .Values.configmap.cnSqlSslCaCert }}
+  sql_client_cert.pem: {{ .Values.configmap.cnSqlSslClientCert }}
+  sql_client_key.pem: {{ .Values.configmap.cnSqlSslClientKey }}
+{{- end }}
```

Comment:
Fix the incorrect annotation reference.

Line 76 references `.Values.config.customAnnotations.secret` but should reference `.Values.customAnnotations.secret` to match the pattern used elsewhere in the template.



Apply this diff to fix the annotation reference:

```diff
-{{- if .Values.config.customAnnotations.secret }}
-{{ toYaml .Values.customAnnotations.secret | indent 4 }}
+{{- if .Values.customAnnotations.secret }}
+{{ toYaml .Values.customAnnotations.secret | indent 4 }}
```